src/picongpu/include/particles/ionization/byField/ADK/AlgorithmADK.hpp

```diff
@@ -82,8 +85,21 @@ namespace ionization
                 float_X dBase = float_X(4.0) * util::cube(protonNumber) / (eInAU * util::quad(nEff)) ;
                 float_X dFromADK = math::pow(dBase,nEff);
 
+                float_X polarizationFactor;
+                /* set up the polarization factor */
+                if(T_linPol)
```

Comment:
This is still a run-time decision thus is not as efficient as possible. 
Just use:
```c++
float_X polarizationFactor = TEMPLATE_PARAM::polarizationFactor;
```
with a struct containing the value **not just a bool**.